src/api/Tron.js

```diff
@@ -0,0 +1,541 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import type {
+  Transaction,
+  TrongridTxInfo,
+  SendTransactionData,
+  SendTransactionDataSuccess,
+  SmartContractTransactionData,
+  FreezeTransactionData,
+  UnfreezeTransactionData,
+  NetworkInfo,
+  BandwidthInfo,
+  SuperRepresentative,
+  SuperRepresentativeData,
+  TronResources
+} from "../families/tron/types";
+import type { Account, SubAccount, Operation } from "../types";
+import {
+  decode58Check,
+  encode58Check,
+  abiEncodeTrc20Transfer,
+  formatTrongridTxResponse,
+  hexToAscii
+} from "../families/tron/utils";
+import { log } from "@ledgerhq/logs";
+import network from "../network";
+import { retry } from "../promise";
+import { makeLRUCache } from "../cache";
+import { getEnv } from "../env";
+import get from "lodash/get";
+import drop from "lodash/drop";
+import sumBy from "lodash/sumBy";
+import take from "lodash/take";
+
+const baseApiUrl: string = getEnv("API_TRONGRID_PROXY");
+
+async function post(url: string, body: Object) {
+  try {
+    const { data } = await network({
+      method: "POST",
+      url,
+      data: body
+    });
+
+    log("http", url);
+
+    if (data.Error) throw new Error(data.Error);
```

Comment:
while trying to quickly integrate this branch with ledger live i got an error "live-common: no network function defined. need to call setNetwork()"
maybe i forgot something but after setting it to the axios library the issue was fixed.

Reply:
Weird, `setNetwork(axios)` is already injected at `tools/src/live-common-setup.js` 🤔

Reply:
we'll soon merge this btw https://github.com/LedgerHQ/ledger-live-common/pull/501 so it will be fixed

Reply:
Please, be careful, in trongrid, the error field from trongrid response is litterally in uppercase => `{ "Error": "...." }`. 
Could you add this case in the #501 PR?